src/solution.jl

```diff
@@ -52,6 +52,10 @@ function solve!(p::Problem, optimizer_factory; silent_solver = false)
         MOI.set(model, MOI.Silent(), true)
     end
 
+    if vexity(p) == NotDcp()
```

Comment:
I think this isn’t quite right actually, we need to check for eg concave minimization problems

Reply:
Or at least, this is right, but there might be other cases.